src/sys/ptrace.rs

```diff
@@ -275,3 +268,207 @@ pub fn cont<T: Into<Option<Signal>>>(pid: Pid, sig: T) -> Result<()> {
     }
 }
 
+#[cfg(target_arch = "x86_64")]
+#[allow(non_camel_case_types)]
+#[derive(Debug, PartialEq)]
+/// Represents all possible ptrace-accessible registers on x86_64
+pub enum Register {
+    R15 = 8 * ::libc::R15 as isize,
```

Comment:
I'd prefer to have all these registers documented, which I think I requested in my earlier review. Could you do this?

Reply:
Please, R15 is just an... R15 register. If someone's using ptrace, it they have to know what the register does anyway.

I don't see any added value in documenting it as 
```
R15 = 8 * ::libc::R15 as isize, /// the R15 register
```

People using `ptrace` are not kids.